tests/pytorch/nightly/fs-transformer.libsonnet

```diff
@@ -196,6 +197,20 @@ local utils = import 'templates/utils.libsonnet';
     },
   },
 
+  local pjrt = tpuVm + experimental.PjRt {
+    tpuSettings+: {
+      tpuVmExtraSetup+: |||
+        pip3 install tqdm
+        git clone -b tpu --single-branch https://github.com/darisoy/fairseq.git fairseq-pjrt/
+      |||,
+    },
+    modelName: 'fs-transformer-pjrt',
+    command: [
```

Comment:
The functional test looks good, but the convergence test looks wrong here:

```
+              'python3' \
+              'fairseq-pjrt/train.py' \
+              'set -u
+              set -e
+              set -x
+              
+              export XLA_USE_BF16=1
+              sudo pip install --editable tpu-examples/deps/fairseq
+              '"'"'python3'"'"' \
+              '"'"'tpu-examples/deps/fairseq/train.py'"'"' \
....
```

Try overriding just the script path: `    paramsOverride+:: {scriptPath: ...`